lib/prmd/templates/schemata/link.md.erb

```diff
@@ -47,7 +47,7 @@
 #### Response Example
 
 ```
-<%- if response_example %>
+<%- if response_example.has_key?('head') %>
```

Comment:
I think we might need to check if it is nil first, then do has_key? Otherwise it will blow up on `has_key?` undefined for nil I think.